Расчет количества книг на дискете.py

```diff
@@ -0,0 +1,16 @@
+# TODO Найдите количество книг, которое можно разместить на дискете
+i = 0
+kol = 0 #кол-во книг
+a = 100
+b = 50
+c = 25
+d = 4
```

Comment:
Верное решение!
Отличное использование цикла!
Но внесу пару корректировок:
Будет немного лучше, если Вы избавитесь от буквенных переменных. Лучше объявляйте более понятные для человека переменные.
Например:
```suggestion
disk_mb = 1.44
pages = 100
lines_per_page = 50
```
И так далее.
Таким образом, ваш код будет более понятен человеку, который не знает суть Вашего задания. Код станет более читаемым